nextcord/abc.py

```diff
@@ -1585,8 +1586,8 @@ def history(
         after: Optional[SnowflakeTime] = None,
         around: Optional[SnowflakeTime] = None,
         oldest_first: Optional[bool] = None,
-    ) -> HistoryIterator:
-        """Returns an :class:`~nextcord.AsyncIterator` that enables receiving the destination's message history.
+    ) -> AsyncIterator[Message]:
+        """Returns an async iterator that enables receiving the destination's message history.
```

Comment:
It may make sense to refer to python's [definition](https://docs.python.org/3/glossary.html#term-asynchronous-iterator) of an async iterator, be it an inline link or a replacement (like `|coro|`, defined in `docs/conf.py`).

Reply:
0f20203